cmd/soroban-cli/src/utils/contract-init-template/contracts/hello_world/Cargo.toml.removeextension

```diff
@@ -1,11 +1,7 @@
 [package]
 name = "soroban-hello-world-contract"
```

Comment:
For `name`, I wonder if it is okay to leave it as `soroban-hello-world-contract` since this is the Cargo.toml for a copy of the [soroban-examples hello_world](https://github.com/stellar/soroban-examples/tree/main/hello_world) contract. 🤔  For reference, we put that contract code into this repo so that users could still user `contract init` even if they didn't have an internet connection.